src/extension/content-script/batteries/GitHub.ts

```diff
@@ -0,0 +1,79 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+const urlMatcher = /^https:\/\/github.com\/([^/]+)(\/([^/]+))?$/;
+
+const battery = (): void => {
+  const urlParts = document.location.pathname.split("/");
+  const username = urlParts[1];
+  const repo = urlParts[2];
+
+  if (username && repo) {
+    handleRepositoryPage(username);
+  } else if (username) {
+    handleProfilePage();
+  }
+};
+
+function parseElement(elementSelector: string) {
+  const text = document.querySelector<HTMLElement>(elementSelector)?.innerText;
+  if (text) {
+    const match = text.match(/(⚡:?|lightning:|lnurl:)\s?(\S+@\S+)/i);
+    if (match) return match[2];
+  }
+}
+
+function handleProfilePage() {
+  const shortBioElement =
+    document.querySelector<HTMLHeadingElement>("[data-bio-text]");
+
+  // This is not a GitHub profile
+  if (!shortBioElement) return;
+
+  const address = parseElement("[data-bio-text]") || parseElement("article");
+
+  if (address) {
+    setLightningData([
+      {
+        method: "lnurl",
+        recipient: address,
+        ...getOriginData(),
+        description:
+          document.querySelector<HTMLHeadingElement>("[data-bio-text]")
+            ?.innerText ?? "",
+        name:
+          document.querySelector<HTMLHeadingElement>('[itemprop="name"]')
+            ?.innerText ?? "",
+        icon:
+          document.querySelector<HTMLImageElement>(`img.avatar-user`)?.src ??
```

Comment:
we need to be more specific here. 

e.g. (`.Layout-sidebar .h-card img.avatar-user`)

the current one matches also other avatars on the page (e.g. the current logged in user)

Reply:
this goes also for the other selectors.

Reply:
I just created a PR that should fix those issues 👍 Sorry for missing that! 😑